src/SingleMovie.js

```diff
@@ -12,14 +11,14 @@ const SingleMovie = ({ singleMovie, clearMovie }) => {
           <button onClick={() => clearMovie()}>GO BACK</button>
         </div>
         <div className="right-side">
-          <h4 className="movie-overview">"Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!"</h4>
-          <h5>"It's a movie!"</h5>
-          <p>Release Date: </p>
-          <p>Run Time: </p>
-          <p>Genres: </p>
-          <p>Avg. Rating: </p>
-          <p>Budget: </p>
-          <p>Revenue: </p>
+          <h4 className="movie-overview">{singleMovie.overview}</h4>
```

Comment:
Good opportunity to destructure the keys out of the singleMove object, so you don't have to type it again and again